spec/features/proposal_spec.rb

```diff
@@ -1,7 +1,8 @@
 require 'spec_helper'
 
 feature Event do
-  let!(:conference) { create(:conference) }
+  let(:registration_period) { create(:registration_period, start_date: Date.current) }
+  let!(:conference) { create(:conference, registration_period: registration_period) }
```

Comment:
Can you please define conference before you define the registration_period? It would be more clear what you are doing, because the logic is: I create a conference, then I create a registration_period for this conference.